Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -41,17 +41,20 @@ private void GridSplitter_Loaded(object sender, RoutedEventArgs e)
                 Element = _gripperDisplay;
             }
 
-            var hoverWrapper = new GripperHoverWrapper(
-                CursorBehavior == SplitterCursorBehavior.ChangeOnSplitterHover
-                ? this
-                : Element,
-                _resizeDirection,
-                GripperCursor,
-                GripperCustomCursorResource);
-            ManipulationStarted += hoverWrapper.SplitterManipulationStarted;
-            ManipulationCompleted += hoverWrapper.SplitterManipulationCompleted;
-
-            _hoverWrapper = hoverWrapper;
+            if (_hoverWrapper == null)
+            {
+                var hoverWrapper = new GripperHoverWrapper(
```

Comment:
Would this underlying instance still need to be updated for this Element or the resize direction if those properties have changed during the unload/reload period?

Reply:
I would have guessed that the wrapper would be updated on Element changes, since it is still attached, even after Unload. But after checking the sources, it looks like Element changes are not tracked at all! So currently, changing the Element after the control is shown would break hover cursor functionality. Thanks for bringing this to my attention. I will submit a new changeset to also fix this issue.

Reply:
And about resizing: The hover wrapper works with PointerEntered/PointerExited of the Element, so it will work regardless of size changes...

Reply:
Please check latest update: Now HoverWrapper is updated when Element changes (no matter if control is currently loaded or unloaded).

Reply:
ping @michael-hawker